src/main/java/bananas/DBConnector.java

```
@@ -0,0 +1,23 @@
+package bananas;
+/**
+ * Created by Bryan on 6/18/2014.
+ **/
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+public class DBConnector {
+	//creates database named ShoppingLists in folder bananas within the users home directory
+	private static final String URL = "jdbc:h2:~/bananas/ShoppingLists";
+	private DBConnector(){
+		//to prevent instantiation of class
+	}
+	public static Connection getConnection(Connection conn){
+		try{
+			conn = DriverManager.getConnection(URL);
+		}catch(SQLException e){
+			e.printStackTrace();
```

This is not handling an exception. 
If a SQLException is ignored here, your program will just break later. 

Never, ever ignore exceptions. 
Exceptions must be handled in the code or passed up to the client caller. 
Ignoring a problem is not a valid way to handle it. 
Ignoring an exception is called 'dropping an exception' 
Exceptions are like eggs. You want to handle them carefully, never drop them, and wash you hands after using them. - Wait, not that last part, but the rest is really true. 
Otherwise, you could make yourself sick spending hours debugging a problem because the issue was carefully ignored somewhere deep in the code. This has happened to be. Well, not the sick part, unless you count sick with rage at the person who 'dropped the exception' and made debugging so much harder. 
Note: sometimes it does make sense to convert the checked exception to a runtime exception. The times this makes sense is when there's no possible way client code can deal with the problem effectively. Note sometimes != always or most of the time. More like once in a while.